### DISCUSSION TITLE
Option to not display front matter

### DISCUSSION THREAD

**Discussion by KieranIRL** [brianpetro/obsidian-smart-connections#173] [Ideas]
I, like most Obsidian users I would assume, use front matter in most of my cards. However, the Smart Connections display on the right of my screen will only show me a preview of the entire note and not the whole note. Because I use a lot of YAML front matter fields, this previews is always just the front matter fields.

![image](https://user-images.githubusercontent.com/129384828/232245775-03674ee3-d51b-41ea-b753-23c4a56b7934.png)

Could you add an option in the settings to not display the front matter in the preview?

Thank you and I seriously love this plugin. Well done mate!

**Comment by winne617** [comment]
for short notes even more useful would be the option not to parse yaml at all

**Reply by dvansoye** [reply]
Agreed. 

I create a new note every morning for my diary. 

I include YAML in my template. 

When I look at the smart connections panel for any of my diary entries, that's all it shows is the YAML (front matter) of other diary entries and not the "meat" of my diary entries.

For example, I often talk about the quality of my sleep.

If the smart connections view showed entries of other days where I had a similar experience (had a good night's sleep, had a bad night's sleep) that would be amazing.

**Comment by krystoflp** [comment]
I would like this too. This PR #144 by @Lisandra-dev looks like this would fix this.
@brianpetro - just making sure you've seen this and the possible fix.

Super plugin by the way!

**Comment by Drakemoor** [comment]
Not displaying it and/or not parsing it will be great. Thanks for the great plugin!

**Reply by Dnarts** [reply]
Yess please! Up!

**Comment by Dnarts** [comment]
why my connectious looks like this? 
![image](https://github.com/user-attachments/assets/3b9bc7e7-8579-410c-9a28-a1e41172d145)

**Reply by Dnarts** [reply]
Up
